modules/adxpremiumAnalyticsAdapter.js

```diff
@@ -19,18 +21,26 @@ const {
   }
 } = CONSTANTS;
 
+let timeoutBased = false;
```

Comment:
is `timeoutBased` somehow configurable? Looking at this adapter I see that it's always false?

Reply:
Currently it is not configurable but we do have plans to make it configurable and that's why we left it in a separate variable.